src/cli/README_HISTORY.md

```diff
@@ -337,6 +339,66 @@ Print the history as a list (last two entries).
 Done
 ```
 
+### router
+
+Usage `history router [list] [<num-entries>]`
+
+Print the route table history. Each item provides:
+
+- Event (`Added`, `Removed`, 'NextHopChnaged', 'CostChanged')
+- Router ID and RLOC16 of router
+- Next Hop (Router ID and RLOC16)
+- Path cost (old `-->` new)
+
+Print the history as a table.
+
+```bash
+> history router
+| Age                  | Event          | ID (RLOC16) | Next Hop    | Path Cost |
++----------------------+----------------+-------------+-------------+-----------+
+|         00:02:53.501 | NextHopChanged | 19 (0x4c00) |           - |         - |
```

Comment:
Indicates next hop changed, but no change in path cost?

Reply:
`-` means we have no next hop and therefore no path cost as well (showing it as `-`).

Reply:
Is it worth showing what the old path cost was?

Reply:
Good point. Will update it to show the change.

Reply:
Updated this in new push.
Another thing I changed is in new push is to output `inf` (instead of `0`) for path cost.